feat(curriculum): added the Tower of Hanoi Python project (#51347)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dc92a9718fb53ca3d1bad7.md

---
id: 64dc92a9718fb53ca3d1bad7
title: Step 9
challengeType: 20
dashedName: step-9
---

# --description--

> The Tower of Hanoi puzzle can be solved in 2<sup>n</sup> - 1 moves, where n is the number of disks. Create a variable named `number_of_moves` and assign it the total number of moves.

Reviewer: ```suggestion
The Tower of Hanoi puzzle can be solved in 2<sup>n</sup> - 1 moves, where n is the number of disks. Declare a variable named `number_of_moves` and assign the total number of moves to this variable.
```
